.github/workflows/test.yml

```diff
@@ -184,6 +184,8 @@ jobs:
         time brew install qemu bash coreutils curl jq
     - name: "Show cache"
       run: ./hack/debug-cache.sh
+    - name: "Inject `no_timer_check` to kernel cmdline"
```

Comment:
Please link a GitHub issue to explain why this is needed on the CI

Reply:
done